packages/amplication-git-utils/src/git/git-client.ts

```diff
@@ -0,0 +1,36 @@
+import { simpleGit, SimpleGit } from "simple-git";
+
+export class GitClient {
+  git: SimpleGit;
+
+  constructor() {
+    this.git = simpleGit();
```

Comment:
Configuration is missing. BaseDir has to be passed to simpleGit function.
It can be seen here in Configuration section:
https://www.npmjs.com/package/simple-git

Reply:
Its not a must